src/main.ts

```diff
@@ -11,7 +11,9 @@ export const options: ts.CompilerOptions = {
 };
 
 export class Minifier {
-  constructor() {}
+  static reservedJSKeywords = {};
+
+  constructor() { Minifier.reservedJSKeywords = Minifier.initReserved(); }
```

Comment:
This means this code is executed every time somebody writes `new Minifier()`. That's bad - it should only happen once, when the program is loaded. Write `static resevedJSKeywords = getReservedKeywords();` instead, so it is only executed once when the program is loaded.